### PR TITLE
feat: wishlist-management

### PR DESCRIPTION
**Pull request by KaustubhM1997**

### CODE REVIEW

**Comment by poornima-alugubelly** [comment]
It would be helpful if u can add description and maybe some images/gif in PR

**Comment by poornima-alugubelly** [comment]
image
Make the banner full width if possible it is looking quite odd this way

File: index.html [hunk 1]
```diff
@@ -35,13 +44,6 @@
...
-
-            <div class="badge-element nav-item">
-                <a href="">
-                    <i class="fa-regular fa-heart icon"></i>
-                </a>
-                <span class="notif-cart">12</span>
-            </div>
         </ul>
     </nav>
     <section class="conatiner-img">
```
Comment (poornima-alugubelly) [inline index.html:49]: there's a typo here for "container"

File: pages/wishlist-management/wishlist-management.css [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    margin: 0;
+
+}
+
+.flex-column {
+
+    display: flex;
+    flex-direction: column;
+}
+
```
Comment (poornima-alugubelly) [inline pages/wishlist-management/wishlist-management.css:17]: maybe u could make this utility class in ur CL so u can use it in other projects as well?

File: pages/wishlist-management/wishlist-management.css [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    top: 0.5rem;
+    font-size: 1.2rem;
+    cursor: pointer;
+
+}
+
+.card-heart-icon:hover {
+
+    color: var(--secondary-color);
+}
```
Comment (poornima-alugubelly) [inline pages/wishlist-management/wishlist-management.css:205]: just a nitpick but there seems to be a lot of extra line breaks in this file

File: pages/wishlist-management/wishlist-management.html [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+    <nav class="navigation-container">
+
+        <div class="nav-brand"> <a class="header-style" href="/index.html">NOZAMA</a>
+            <i class="fa-solid fa-arrow-left-long"></i>
+        </div>
+
+        <div class="input-search">
+
+            <input class="search" placeholder="Search" type="text">
+        </div>
```
Comment (poornima-alugubelly) [inline pages/wishlist-management/wishlist-management.html:28]: maybe u could use input type="search" here for better accessibility

**Review by poornima-alugubelly — APPROVED** [review]
Good work!

File: pages/product-listing/product-listing.css [hunk 5]
```diff
@@ -120,6 +115,7 @@ input {
     grid-gap: 1.5rem;
     grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
     margin-top: 0.5rem;
+    margin-top: 1rem;
 }
```
Comment (BelideNikhil) [inline pages/product-listing/product-listing.css:119]: Either use gap or margin, I don't think u need both.
GAP would be better as it's a grid

**Review by srishti-maurya — APPROVED** [review]
looks good

File: pages/wishlist-management/wishlist-management.css [hunk 6]
```diff
@@ -0,0 +1,205 @@
+@import url(/main.css);
+
+* {
+
+
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+
+}
```
Comment (BelideNikhil) [inline pages/wishlist-management/wishlist-management.css:10]: Reset is not needed if you have a reset specified at the start/ main css file

**Review by BelideNikhil — APPROVED** [review]
I see that you already have utils for flex
try creating more of them as I see you have written a lot of flex in css

File: pages/product-listing/product-listing.css (hunk 5 above)
Comment (KaustubhM1997) [reply pages/product-listing/product-listing.css:119]: Hi Nikhil, the margin is for the space between the nav and the heading "show all products"

**Pull request merged**